EIPS/eip-4834.md

```diff
@@ -0,0 +1,224 @@
+    /// @param      subdomain The subdomain to set
+    function setDomain(string memory name, address subdomain) external;
+
+    /// @notice     Delete the subdomain with a given name
+    /// @dev        This should revert if the domain doesn't exist or if
+    ///             `canDeleteDomain(msg.sender, name, this)` is `false`
+    /// @param      name The subdomain to delete
+    function deleteDomain(string memory name) external;
+
+
+    //// Parent Domain Access Control
+
+    /// @notice     Get if an account can create a subdomain with a given name
+    /// @dev        This must return `false` if `hasDomain(name)` is `true`.
+    ///             If `subdomain` is a domain, this should return `false` if
+    ///             `subdomain.canPointSubdomain(msg.sender, name, this) is `false`.
+    /// @param      updater The account that may or may not be able to create/update a subdomain
+    /// @param      name The subdomain name that would be created/updated
+    /// @param      subdomain The subdomain that would be set
+    /// @return     Whether an account can update or create the subdomain
+    function canCreateDomain(address updater, string memory name, address subdomain) external view returns (bool);
+
+    /// @notice     Get if an account can update or create a subdomain with a given name
+    /// @dev        This must return `false` if `hasDomain(name)` is `false`.
+    ///             If `subdomain` is a domain, this should return `false` if
+    ///             `subdomain.canPointSubdomain(msg.sender, name, this) is `false`.
+    ///             If `getDomain(name)` is also a domain, this should return `false` if
+    ///             `getDomain(name).canMoveSubdomain(msg.sender, this, subdomain)` is `false`.
+    /// @param      updater The account that may or may not be able to create/update a subdomain
+    /// @param      name The subdomain name that would be created/updated
+    /// @param      subdomain The subdomain that would be set
+    /// @return     Whether an account can update or create the subdomain
+    function canSetDomain(address updater, string memory name, address subdomain) external view returns (bool);
+
+    /// @notice     Get if an account can delete the subdomain with a given name
+    /// @dev        This must return `false` if `hasDomain(name)` is `false`.
+    ///             If `getDomain(name)` is a domain, this should return `false` if
+    ///             `getDomain(name).canDeleteSubdomain(msg.sender, this, subdomain)` is `false`.
+    /// @param      updater The account that may or may not be able to delete a subdomain
+    /// @param      name The subdomain to delete
+    /// @return     Whether an account can delete the subdomain
+    function canDeleteDomain(address updater, string memory name) external view returns (bool);
+
+
+    //// Subdomain Access Control
+
+    /// @notice     Get if an account can point this domain as a subdomain
```

Comment:
I'm not sure what this means. Would you mind expanding it a little bit?

Reply:
I've rephrased it: `Get if an account can set this domain as a subdomain`